developer-docs-site/docs/tutorials/your-first-nft-token-v2.md

```diff
@@ -0,0 +1,261 @@
+--- 
+title: "Mint NFT with the token v2 SDKs"
+slug: "your-first-nft-token-v2"
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import ThemedImage from '@theme/ThemedImage';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Mint Token v2 NFTs with the Aptos SDKs
+
+This tutorial describes how to create and transfer NFTs on the Aptos blockchain. The Aptos implementation for core NFTs can be found in [aptos_token.move](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-token-objects/sources/aptos_token.move) Move module.
+
+## Step 1: Pick an SDK
+
+Install your preferred SDK from the below list:
+
+* [TypeScript SDK](../sdks/ts-sdk/index.md)
+* [Python SDK](../sdks/python-sdk.md)
+
+---
+
+## Step 2: Run the example
+
+Each SDK provides an `examples` directory. This tutorial covers the `simple-nft-v2` example. 
+
+Clone the `aptos-core` repo: 
+```bash 
+git clone git@github.com:aptos-labs/aptos-core.git ~/aptos-core
+```
+
+<Tabs groupId="sdk-examples">
+    <TabItem value="python" label="Python">
+
+Navigate to the Python SDK directory:
+  ```bash
+  cd ~/aptos-core/ecosystem/python/sdk
+  ```
+
+Install the necessary dependencies:
+  ```bash
+  curl -sSL https://install.python-poetry.org | python3
+  poetry update
+  ```
+
+Run the Python [`simple-nft`](https://github.com/aptos-labs/aptos-core/blob/main/ecosystem/python/sdk/examples/simple-nft-v2.py) example:
+  ```bash
+  poetry run python -m examples.simple-nft-v2 main
+  ```
+  </TabItem>
+</Tabs>
+
+---
+
+## Step 3: Understand the output
+
+The following output should appear after executing the `simple-nft-v2` example, though some values will be different:
+
+```yaml
+=== Addresses ===
+Alice: 0x5ce75334cd71793f417cc76cbb0b191660b8868538d6590086f4f231c73ed073
+Bob: 0x443b51f29aa0beb2387e24752710e3e4edabd69b4bf336db333e802ac5a59f97
+
+  === Initial Coin Balances ===
+Alice: 100000000
+Bob: 100000000
+
+  === Creating Collection and Minting Token ===
+Alice's collection: ReadObject
+  0x1::object::ObjectCore: Object[allow_ungated_transfer: False, owner: 0x5ce75334cd71793f417cc76cbb0b191660b8868538d6590086f4f231c73ed073]
+  0x4::collection::Collection: AccountAddress[creator: 0x5ce75334cd71793f417cc76cbb0b191660b8868538d6590086f4f231c73ed073, description: Alice's simple collection, name: Alice's, ur: https://aptos.dev]
```

Comment:
is that right "ur" at the end, can you fix the bug in the text and/or the sdk?

Reply:
we also need to make sure both python and typescript output this content consistently :grimacing: